docs/functional_api.rst

```diff
@@ -1,3 +1,5 @@
+.. _functional api:
```

Comment:
What is that for?

Reply:
So I can link to it between pages (from the math.rst).